lib/ansiblelint/formatters/__init__.py

```diff
@@ -1,20 +1,40 @@
 """Output formatters."""
+import os
+
 try:
     from ansible import color
 except ImportError:
     from ansible.utils import color
-from ansiblelint.utils import normpath
 
 
-class Formatter(object):
+class BaseFormatter(object):
```

Comment:
Now that we dropped Python 2, no need to explicitly make classes new-style.
```suggestion
class BaseFormatter:
```